ledger/ledger.go

```diff
@@ -160,6 +160,11 @@ func OpenLedger(
 		l.genesisAccounts = make(map[basics.Address]basics.AccountData)
 	}
 
+	l.blockQ, err = bqInit(l)
```

Comment:
how about renaming `bqInit` to `makeBlockQueue` to make it clear this is called once and creates the object.